testsuite/MDAnalysisTests/coordinates/data/test_topology.pdb

```diff
@@ -0,0 +1,8 @@
+MODEL
+CRYST1 80.000   80.000   80.000  60.00  60.00  90.00
+ATOM      1 N    MET     1      00.000  00.000  00.000  1.00 84.71
```

Comment:
instead of N make it CA... then it looks at least like a CA representation of something...